### PR TITLE
Add NEMO

### PR DESCRIPTION
**Pull request by dylan-klomparens**
# Awesome NIST Guide to Awesomeness

## Baseline Checklist *(prerequisite)*

The repository complies with our [Terms of Use][_ntos].

- [x] README.md or README.rst clearly describes, in layman's terms:
  - [x] goals of the repository and project
  - [x] whether the code is usable (pre-release, production-ready, or in-between?)
  - [x] how to get in touch with the development team
  - [x] acknowledges collaborators and reused code, if applicable
- [x] Documentation exists, in README.md or separately, and explains
  - [x] dependencies, if any
  - [x] installation process
  - [x] basic usage
- [x] LICENSE.md or LICENSE.txt dedicates the software to the public domain using
      [approved language][_dirl], preferably a duplicate of [LICENSE.md][_stdl]

## Maturity Checklist *(bonus)*

The repository demonstrates mature development practices, including
integration with services that improve maintainability and an overall
commitment to helping newcomers quickly achieve proficiency.

- [ ] Developers leverage GitHub as a communication tool
  - [x] Description, Website, and Topics fields are meaningfully populated
  - [ ] Closed pull requests indicate a [branching][_brwf] or otherwise
        consistent workflow
  - [ ] Merged pull requests underwent formal [code review][_ghrv]
  - [ ] Badges in README.md display positive status messages
  - [ ] CONTRIBUTING.md provides clear guidance on
    - [ ] where the stable code resides, if not on `master`
    - [ ] expected workflow and release process (rolling, [semver][_smvr], or other)
- [ ] Self-documenting code with link to online docs
  - [ ] using [Doxygen][_doxy], [Docstrings][_docs], or similar
  - [ ] hosted by [NIST][_nstp], [readthedocs][_rtfd], or similar
  - [ ] provides worked examples and tutorials
- [ ] Tests automatically run on a continuous integration service,
      *e.g.* [Travis CI][_trvs], [Circle][_crcl], or similar
- [x] Insights tab on GitHub indicates a healthy software environment
  - [x] Contributors shows a history of commits (not just one)
  - [x] Dependency Graph does not flag security vulnerabilities
- [x] Git branches are used appropriately

## Further Justification

In addition to the generic items selected above, this repository is *awesome*
because...
* I gave a talk about NEMO at the [UGIM 2018 conference](https://ugim.nano.upenn.edu/).
* Princeton University has committed to using it for their NanoFab lab management; they have the prototype instance up and running, and are in the process of migrating to roll out a production instance later this year. I am optimistic that other NanoFabs will also adopt NEMO, as some have verbally expressed interest.

[_brwf]: https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows
[_crcl]: https://circleci.com
[_dirl]: https://www.nist.gov/director/licensing
[_docs]: https://www.python.org/dev/peps/pep-0257/
[_doxy]: https://www.stack.nl/~dimitri/doxygen
[_ghrv]: https://help.github.com/articles/about-pull-request-reviews/
[_nstp]: https://pages.nist.gov
[_ntos]: http://odiwiki.nist.gov/ODI/GitHubFAQ#Public_Repository_Guidelines
[_rtfd]: https://readthedocs.org
[_smvr]: https://semver.org
[_stdl]: https://github.com/usnistgov/awesome-nist/blob/master/LICENSE.md
[_trvs]: https://travis-ci.org

## REPOSITORY NAME

I have reviewed REPOSITORY_NAME, and consider it awesome.

## Baseline Requirements

Paste [checklist](../AWESOME.md) here.

## Subjective Opinion

Write your thoughts down.

### CODE REVIEW

**Comment by tkphd** [comment]
## Baseline Checklist *(prerequisite)*

The repository complies with our [Terms of Use][_ntos].

- [x] README.md or README.rst clearly describes, in layman's terms:
  - [x] goals of the repository and project
  - [x] whether the code is usable (pre-release, production-ready, or in-between?)
  - [ ] how to get in touch with the development team
  - [x] acknowledges collaborators and reused code, if applicable
- [x] Documentation exists, in README.md or separately, and explains
  - [x] dependencies, if any
  - [x] installation process
  - [x] basic usage
- [x] LICENSE.md or LICENSE.txt dedicates the software to the public domain using
      [approved language][_dirl], preferably a duplicate of [LICENSE.md][_stdl]

## Maturity Checklist *(bonus)*

The repository demonstrates mature development practices, including
integration with services that improve maintainability and an overall
commitment to helping newcomers quickly achieve proficiency.

- [x] Developers leverage GitHub as a communication tool
  - [x] Description, Website, and Topics fields are meaningfully populated
  - [x] Closed pull requests indicate input from collaborators
  - [ ] Merged pull requests underwent formal [code review][_ghrv]
  - [ ] Badges in README.md display positive status messages
  - [ ] CONTRIBUTING.md provides clear guidance
- [ ] Self-documenting code with link to online docs
- [ ] Tests automatically run on a continuous integration service,
      *e.g.* [Travis CI][_trvs], [Circle][_crcl], or similar
- [x] Insights tab on GitHub indicates a healthy software environment
  - [x] Contributors shows a history of commits (not just one)
  - [x] Dependency Graph does not flag security vulnerabilities
- [x] Git branches are used appropriately

## Further Justification

In addition to the generic items selected above, this repository is *awesome*
because the README uses prose and imagery to clearly explain how to use NEMO and what to expect, as well as Docker files and a DockerHub instance for visitors and users to quickly and easily get started.

## Bottom Line

While testing would help improve maintainability and might catch bugs, this repository is a good example of how to use GitHub to document, motivate, and distribute code. It is awesome.


[_brwf]: https://git-scm.com/book/en/v2/Git-Branching-Branching-Workflows
[_crcl]: https://circleci.com
[_dirl]: https://www.nist.gov/director/licensing
[_docs]: https://www.python.org/dev/peps/pep-0257/
[_doxy]: https://www.stack.nl/~dimitri/doxygen
[_ghrv]: https://help.github.com/articles/about-pull-request-reviews/
[_nstp]: https://pages.nist.gov
[_ntos]: http://odiwiki.nist.gov/ODI/GitHubFAQ#Public_Repository_Guidelines
[_rtfd]: https://readthedocs.org
[_smvr]: https://semver.org
[_stdl]: https://github.com/usnistgov/awesome-nist/blob/master/LICENSE.md
[_trvs]: https://travis-ci.org

File: README.md [hunk 1]
```diff
@@ -23,6 +23,7 @@ a look!
 
 * [ChebTools](https://github.com/usnistgov/ChebTools) - C++ and Python tools
   for working with Chebyshev expansions.
+* [NEMO](https://github.com/usnistgov/NEMO) - a laboratory logistics web application. Use it to schedule reservations, control tool access, track maintenance issues, and more.
```
Comment (tkphd) [inline README.md]: Please reduce summary to fit on one line, in one sentence if possible.

**Review by tkphd — CHANGES_REQUESTED** [review]

**Comment by dylan-klomparens** [comment]
I made the description into a single sentence. It might be difficult to take away more, while still conveying what the program does. Open to suggestions if you can think of improvements.

I also wanted to note that there is a section in the README that tells people how to get in touch with the developers (which is just me right now). Perhaps it should be more prominent or explicit? What do you think?

> The community page outlines how to ask questions and contribute to NEMO. Bugs can be reported to the issues page. If you've found a security issue with NEMO then please read our security policy and tell us discretely.

**Comment by tkphd** [comment]
I made slight edits, so that the NEMO description fits on a single line in the rendered Markdown.

Mention of the community page, and in turn the references to the mailing list available there, are sufficient for the purpose of this list. It's up to you whether to make it more prominent. Your README is not at the "dangerously long" point yet, so moving things from the Wiki into the README is not crazy; but it might be part of a concerted effort to make the software self-documenting, rather than a piecemeal migration of some content.

If you're looking for advice, consider filing an issue on [awesome-nist][_awsm], [USNISTGOV][_usng], or [discuss][_dscs], or in the [Open-source interest group][_osig].


[_awsm]: https://github.com/usnistgov/awesome-nist/issues
[_dscs]: https://github.com/usnistgov/discuss/issues
[_osig]: https://groups.google.com/a/list.nist.gov/d/forum/o10sig
[_usng]: https://github.com/usnistgov/USNISTGOV/issues

**Review by tkphd — APPROVED** [review]

**Comment by tkphd** [comment]
Merged in 4937665b8f88f09edb3ea67e22cde58f7bd3be81

**Pull request closed without merging**